src/ResourceManager/Compute/Commands.Compute.Test/packages.config

```diff
@@ -7,7 +7,7 @@
   <package id="Microsoft.Azure.Graph.RBAC" version="1.9.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="11.3.1-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="3.1.0-preview" targetFramework="net45" />
```

Comment:
We will need to update this with a new package version, after we publish the SDK changes